test/profiler/test_profiler.py

```diff
@@ -70,6 +70,7 @@
     TEST_WITH_ROCM,
     TestCase,
 )
+from torch.testing._internal.triton_utils import requires_gpu
```

Comment:
This API is used in Inductor, especially for triton, so I think we should avoid it here.